frontend/src/editor/player.tsx

```diff
@@ -160,3 +163,35 @@ export function PlayerBar({ documentId, editor }: { documentId: string; editor:
 export function startTimeToClassName(startTime: number) {
   return `start-${startTime.toFixed(3).replace('.', '')}`;
 }
+
+function PlaybackSpeedDropdown({ onChange }: { onChange: (v: number) => void }) {
+  const possibleRates = [0.5, 0.7, 1.0, 1.2, 1.5, 1.7, 2.0];
+
+  const [value, setValue] = useState(1.0);
+
+  return (
+    <Dropdown
```

Comment:
IMHO a classic `<select`-attribute would be better here. For example it also provides an indication of the currently selected element, support for pressing <kbd>Esc</kbd>, ...

Reply:
good point